addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/internal/messages/RFXComCurrentMessage.java

```diff
@@ -0,0 +1,217 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.rfxcom.internal.messages;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.eclipse.smarthome.core.library.items.NumberItem;
+import org.eclipse.smarthome.core.library.types.DecimalType;
+import org.eclipse.smarthome.core.types.State;
+import org.eclipse.smarthome.core.types.Type;
+import org.eclipse.smarthome.core.types.UnDefType;
+import org.openhab.binding.rfxcom.RFXComValueSelector;
+import org.openhab.binding.rfxcom.internal.exceptions.RFXComException;
+
+/**
+ * RFXCOM data class for current message.
+ *
+ * @author Unknown - Initial contribution
+ * @author Pauli Anttila - for the Similar RFXComEnergyMessage code
+ * @author Jordan Cook - Added support for CURRENT devices, such as OWL CM113
+ */
+public class RFXComCurrentMessage extends RFXComBaseMessage {
+
+    public enum SubType {
+        ELEC1(0),
```

Comment:
I think you have minor mistake with the bytes of the SubTypes, see also my PR #1247 & PR # 4629 for openhab1

According to the manual of the RFXCom the only valid SubType is ELEC1 (with byte 1), the only way to use this properly with the current code (it use the list of values instead of actual byte when parsing the subtype) is adding some placeholder in front of it, eg UNSUPPORTED(0) followed by ELEC1(1)